addons/mail/static/src/components/chat_window_manager/chat_window_manager.xml

```diff
@@ -3,18 +3,20 @@
 
     <t t-name="mail.ChatWindowManager" owl="1">
         <div class="o_ChatWindowManager">
-            <t t-if="env.messaging.isInitialized">
-                <!-- Note: DOM elements are ordered from left to right -->
-                <t t-if="env.messaging.chatWindowManager.hasHiddenChatWindows">
-                    <ChatWindowHiddenMenu class="o_ChatWindowManager_hiddenMenu"/>
-                </t>
-                <t t-foreach="env.messaging.chatWindowManager.allOrderedVisible" t-as="chatWindow" t-key="chatWindow.localId">
-                    <ChatWindow
-                        chatWindowLocalId="chatWindow.localId"
-                        hasCloseAsBackButton="env.messaging.device.isMobile"
-                        isExpandable="!env.messaging.device.isMobile and !!chatWindow.thread"
-                        isFullscreen="env.messaging.device.isMobile"
-                    />
+            <t t-if="env.messaging">
+                <t t-if="env.messaging.isInitialized">
```

Comment:
This could be a single condition on this template because none of those have `t-else`.

Reply:
Resolved but not done/commented?

Reply:
Indeed, made changes in discuss and messaging menu, forgot chat window manager. Will do.